### PR TITLE
[Terra Form single select] Announces the expansion state Upon revisiting

### PR DESCRIPTION
**Pull request by gt106551**
### Summary
Fixed issue - When selecting and pressing Enter in a dropdown, it announces the expansion state Upon revisiting, it shouldn't reiterate the expanded status.

**What was changed:**

Removed "Expanded combobox" string when navigates to other items on the list.

**Why it was changed:**

"Expanded combobox" text was reading by Voice over when navigates to the first item from other list items

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [x] Visual testing (please attach a screenshot or recording)
        **Before Changes:**

https://github.com/cerner/terra-core/assets/126649018/5311702d-9da2-45c9-86b1-c890bce6e762

         **After Changes:**
https://github.com/cerner/terra-core/assets/126649018/eab473e7-64df-49d3-a846-74be784320ac

- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9672

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by supreethmr** [comment]
I am able to see Expanded text on aria-label of first item even after navigating through all the other items. was these code changes tested with screen reader.?

https://github.com/cerner/terra-core/assets/12869809/5601e390-73de-4481-bf31-2ff0192f20d5

**Comment by gt106551** [comment]
> I am able to see Expanded text on aria-label of first item even after navigating through all the other items. was these code changes tested with screen reader.?
> 
>  Screen.Recording.2023-09-25.at.12.40.59.PM.mov

I have tested with all browsers(VO&JAWS), it is working as expected.

File: packages/terra-form-select/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Added
+ * Fixed it announces the expansion state Upon revisiting.
```
[the remark this answers is not in the document]
Comment (gt106551) [reply packages/terra-form-select/CHANGELOG.md]: Updated

File: packages/terra-form-select/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+ * Fixed it announces the expansion state Upon revisiting.
```
Comment (sdadn) [inline packages/terra-form-select/CHANGELOG.md:6]:
```suggestion
 * Fixed screenreaders announcing the expansion state upon revisit.
```

Comment (gt106551) [reply packages/terra-form-select/CHANGELOG.md]: Updated

**Comment by supreethmr** [comment]
This change is already being taken care on PR : https://github.com/cerner/terra-core/pull/3855 aria-label has been removed and text has been added to visually hidden text.

**Pull request closed without merging**